### PR TITLE
Add note that USING PERIODIC COMMIT does not work with SSR

### PR DESCRIPTION
**Pull request by tobias-johansson**

### CODE REVIEW

**Comment by tobias-johansson** [comment]
[Document that PERIODIC COMMIT cannot be used with server-side routing](https://trello.com/c/ui6M54TD/556-document-that-periodic-commit-cannot-be-used-with-server-side-routing)

File: cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/UsingTest.scala [hunk 1]
```diff
@@ -233,7 +233,12 @@ class UsingTest extends DocumentingTest {
...
       }
       note {
-        p("The <<query-use, `USE` clause>> can not be used together with the `PERIODIC COMMIT` clause.")
+        p("The <<query-use, `USE` clause>> can not be used together with the `PERIODIC COMMIT` query hint.")
+      }
+      note {
+        p(
+          """Queries with the `PERIODIC COMMIT` query hint can not be routed by <<operations-manual#causal-clustering-routing, Server-side routing>>.
+            |Such queries must rely on standard client-side routing, done by the Neo4j Driver.""".stripMargin)
       }
```
Comment (Hunterness) [inline cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/UsingTest.scala:242]: Shouldn't this note also be in the periodic commit section (generated from `LoadCSVTest`)

Comment (tobias-johansson) [reply cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/UsingTest.scala:242]: Nice catch. I added a line there as well.

**Review by hedengran — APPROVED** [review]

**Review by Hunterness — APPROVED** [review]

**Comment by MarkWoulfeNeo4j** [comment]
LGTM

**Pull request merged**